ethpm/backends/ipfs.py

```diff
@@ -154,7 +154,7 @@ def base_uri(self) -> str:
 MANIFEST_URIS = {
     "ipfs://QmVu9zuza5mkJwwcFdh2SXBugm1oSgZVuEKkph9XLsbUwg": "standard-token",
```

Comment:
I'm curious why these didn't change?

Reply:
Ahh good catch, well it's because this is just a small cache for the `DummyIPFSBackend` and `DummyIPFSBackend.fetch_uri_contents()` simply fetches the local asset rather than fetching it from IPFS, so the hash isn't really used in any meaningful way. But, they're all updated now.